e2core/server/handlers.go

```diff
@@ -72,140 +76,7 @@ func (s *Server) executePluginByNameHandler() echo.HandlerFunc {
 
 		responseData := seq.Request().State[mod.FQMN]
 
-		return c.Blob(http.StatusOK, "application/octet-stream", responseData)
-	}
-}
-
-func (s *Server) executePluginByRefHandler(l zerolog.Logger) echo.HandlerFunc {
```

Comment:
removed because the code path no longer exists. This was only called when admin was NOT enabled, but since we don't do that here no more, this is dead code